include/logInFunctions.php

```diff
@@ -0,0 +1,71 @@
+
+<script src="/include/jquery.js"></script>
+<script type='text/javascript'>
+
+function togglePasswordVisibility(){
+	var x = document.getElementById("myInput");
+    if (x.type === "password") {
+        x.type = "text";
+    } else {
+        x.type = "password";
+    }
+}
+
+</script>
+
+
+<?php
+
+
+
+function validateFormField($formField){
+	global $Errors;
+	if(!$_REQUEST[$formField]){
+		$Errors[$formField] = 'required';
+	}
+}
+
+function createUser($username, $password){
+	insertUser($username, $password);
+	header("Location: accountCreated.php"); // this is how you redirect the browser directly.
+	exit();
+}
+
+
+function insertUser($username, $password){
+	$result = dbQuery("
+		INSERT INTO users(username, password)
+		VALUES(:username, :password)
+	", array(
+		'username'=>$username,
+		'password'=>$password
+	))->fetchAll(); //All
+}
+
+function getUserRow($username){
+	$result = dbQuery("
+		SELECT *
+		FROM users
+		WHERE username = :username
+	", array(
+		'username'=>$username
+		))->fetch();
+	return $result;
+}
+
+function verifyUser($username, $password){
+
+//if this user exists w this pass in the database, then return true
+//if not, return false
+
+	$result=getUserRow($username);
```

Comment:
This hopefully won't matter if you take my suggestion from above and validate that a username isn't already in use when someone signs up, but this function is an example of why this matters. Right now, if two users have the same username, you're just getting one of them here and checking against that password. So someone could potentially log in with the correct username/password combo and still not get logged in.

You asked if this is the correct way to verify a user. This definitely works for now, but later in the summer I'm going to go over some more advanced password security stuff. You'll want to make some adjustments after that.